### DISCUSSION TITLE
Expanded FLARE Campaign, Tech Demo for New Features. ~1.16-17+?

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#916] [Ideas]
So there are so many new features coming to FLARE (new itemization, statistics, attributes, creature encounters etc.) that I am wondering if an expansion to the FLARE campaign should be in order?

As in to show people how things can be set up and serve as working examples and templates for features in FLARE and have ways for people to reverse engineer features for their own projects. I was thinking after 1.15 / all of the bigger features under development are implemented, bug fixed / working properly. I am wondering if that kind of potential project is interesting / worth it for people?

Any feedback, thoughts, ideas and opinions?

**Comment by WuShufei** [comment]
I just happened across Flare Empyrean Campaign in repo and started playing it last night.  Hooked!  I do wish a "pause" panel with keybindings were obvious in game.  But really, this campaign is so fun, lucid, and interesting, I wish it were more well known, maybe turned into a MMRPG even.  Thanks heaps, devs!

**Comment by WuShufei** [comment]
PS, I'm playing on an overwhelmed Raspi 4B in a Raspad.  Good choices on touchscreen and keyboard both.  And happy to say the engine plays well on armhf.
